sw/device/tests/entropy_src_csrng_test.c

```diff
@@ -205,14 +204,30 @@ void ottf_external_isr(void) {
 bool test_main(void) {
   init_peripherals();
 
+  // Get test random parameters before we disable the entropy complex.
```

Comment:
ah nice... i actually wonder if we should move this into testutil and do this for all the entropy complex tests...

Reply:
sg. I can look into that

Reply:
o no i don't mean we need to do it here, just suggesting that this may be good to do elsewhere.

Reply:
SG :)